src/layouts/_header.scss

```diff
@@ -0,0 +1,27 @@
+.navigation {
+  float:right;
+  display: block;
+  margin-right: 15px;
```

Comment:
You're hardcoding some `px` in few places. Would we benefit from the use of the variables like `$govuk-spacing-scale-3`?

Reply:
Was trying to do it fast as thought we needed it out yesterday for the deploy story, but if there's more scope can go and find the mixins.